spec/schema.json

```diff
@@ -410,6 +416,21 @@
         "SUBMITTED",
         "PROCESSED"
       ]
+    },
+    "Ineligible": {
+      "type": "object",
+      "properties": {
+        "status": {
+          "$ref": "#/definitions/IneligibilityStatus"
+        }
+      }
+    },
+    "IneligibilityStatus": {
```

Comment:
I approved but spotted something, the INELIGIBLE_XXXX status should be added to 
"Status": {
      "type": "string",
      "enum": [
        "OPEN",
        "SUBMITTED",
        "PROCESSED"
      ]
    },

so we don't need a new class of statuses

Reply:
Removed new class added to old class.